apps/console/src/features/identity-providers/components/settings/authenticator-settings.tsx

```diff
@@ -446,6 +446,61 @@ export const AuthenticatorSettings: FunctionComponent<IdentityProviderSettingsPr
         );
     };
 
+    /**
+     * A predicate that checks whether a give federated authenticator
+     * is a default authenticator.
+     * @param auth
+     * @returns true if {@code auth.data.isDefault} is truthy
+     */
+    const _isDefaultAuthenticatorPredicate = (
+        auth: FederatedAuthenticatorWithMetaInterface
+    ): boolean => {
+        if (auth.data == void 0) { // Safe unwrap
```

Comment:
Use optional chaining rather than the `void` operator. 
```suggestion
        if (!auth?.data) {
           return false;
        }
```

Reply:
Applied this code style improvement in commit ba81c1a